indico/modules/events/reminders/templates/reminders.html

```diff
@@ -11,7 +11,12 @@
         {% for reminder in reminders %}
             <li>
                 <strong>
-                    <a href="{{ url_for('.edit', reminder) }}">
+                    <a href="#" data-href="{{ url_for('.edit', reminder) }}"
+                        {% set button_title =_("Reminder details") if reminder.is_sent else _("Edit reminder") %}
```

Comment:
i'd put this before the `<a>` tag. but with the ternary operator it's now short enough to just put it in the next line without assigning it to a variable first